include/openssl/ec.h

```diff
@@ -363,6 +363,10 @@ EC_GROUP *EC_GROUP_new_from_ecpkparameters(const ECPKPARAMETERS *params);
 ECPKPARAMETERS *EC_GROUP_get_ecpkparameters(const EC_GROUP *group,
                                             ECPKPARAMETERS *params);
 
+/* TODO */
```

Comment:
Given that the timing properties and general validity (order must be a prime for ECDSA) of this function are non-trivial and public APIs are forever, it seems better to keep this private.

OpenSSL's public API surface is unnecessarily huge as it is, and exposes too many internals. Our repeated experience in BoringSSL is that the OpenSSL API often gets in the way of fixing side channels and other security problems in the library. The stability requirements make it difficult to fix this in the short term, but let's refrain from adding to the problem unnecessarily.

Reply:
> OpenSSL API often gets in the way of fixing side channels

I have the same experience.

Then to me, `ec_lib.c` is not the right place for `EC_GROUP_do_inverse_ord`. Where do you suggest it should go? (IMHO simply omitting the function prototype from the header is insufficient. Something something opaque API change.)